staging/src/k8s.io/apimachinery/pkg/conversion/unstructured/converter.go

```diff
@@ -465,6 +524,25 @@ func pointerToUnstructured(sv, dv reflect.Value) error {
 	return toUnstructured(sv.Elem(), dv)
 }
 
+func isZero(v reflect.Value) bool {
+	switch v.Kind() {
+	case reflect.Array, reflect.String:
+		return v.Len() == 0
+	case reflect.Bool:
+		return !v.Bool()
+	case reflect.Int, reflect.Int8, reflect.Int16, reflect.Int32, reflect.Int64:
+		return v.Int() == 0
+	case reflect.Uint, reflect.Uint8, reflect.Uint16, reflect.Uint32, reflect.Uint64:
+		return v.Uint() == 0
+	case reflect.Float32, reflect.Float64:
+		return v.Float() == 0
+	case reflect.Map, reflect.Slice, reflect.Ptr, reflect.Interface:
+		// TODO: It seems that 0-len maps are ignored in it.
```

Comment:
Some tests are failing when checking "object -> map[string]interface{}" conversion is on.

It seems related to empty maps, but didn't yet figure it out.